core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -461,6 +461,36 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }
   }
 
+  def updateBillingProjectBillingAccount(billingProjectName: RawlsBillingProjectName, updateAccountRequest: UpdateRawlsBillingAccountRequest): Future[Unit] = {
```

Comment:
`v2_workspaces` returns a `PerRequestMessage` but I'd like to stop proliferating that pattern so I don't use that here. If y'all would find it easier to mesh with, I am happy to make this return a `PerRequestMessage` as well and then take care of it all in my on-going [PerRequest refactor](https://github.com/broadinstitute/rawls/pull/1408) after PPW goes live.

Reply:
Out of curiosity, what's the rationale in removing that pattern?

Reply:
The main reason is that it obfuscates the return types of many of the methods in the service layer, which can make the code harder to reason about and test. It also causes status codes to bleed more deeply into the business logic when preferably that would be taken care of in the API route layer.